config/routes.rb

```diff
@@ -14,6 +14,7 @@
   resources :users
   resources :duties do
     collection do
+      get 'dropped', to: 'duties#dropped'
```

Comment:
Is this action in the controller alrd?

Reply:
No :( It's guesswork since the duty grab/drop branch hasn't been pushed

Reply:
It's for the url sent in the emails

Reply:
I see. I thought it'll be better if you put a placeholder in the email instead